exercises/rotational-cipher/example.rs

```diff
@@ -0,0 +1,16 @@
+pub fn rotate(text: & 'static str, shift_key: u8) -> String {
+
+    return text.chars().map(|c| {
```

Comment:
how about eliding the `return` and semicolon? I don't know if there is some Rust style guide that demands it, but at least https://github.com/Manishearth/rust-clippy/wiki#needless_return asks for it.